### DISCUSSION TITLE
Video support?

### DISCUSSION THREAD

**Discussion by NanoscaleSimulations** [reflex-dev/reflex#665] [Q&A]
Hello there,

I am building an app using pynecone. I want to show a video demo of how the app works on the front page, but i cannot seem to find any components for showing videos. Is this not possible at the moment? If not, is there a way to work around it? And will it be possible at some point?

Kind regards
Zan

**Comment by FHU-yezi** [comment]
Maybe you can use the `pc.html` Component and put video elements.
